packages/wast-parser/src/grammar.js

```diff
@@ -448,7 +442,7 @@ export function parse(tokensList: Array<Object>, source: string): Program {
      */
     function parseBlock(): BlockInstruction {
       let label = t.identifier(getUniqueName("block"));
-      let blockResult;
+      let blockResult = null;
```

Comment:
The default shoud probably be `0x40` it's null in the blockTypes

Reply:
@xtuc just realised I didn't address this comment? Would you prefer ...

```
import {blockTypes }  from '@webassembljs/helper-wasm-bytecode';
...
let blockResult = blockTypes.0x40;
```

Feels like an odd dependency to me ... wast-parser depending on helper-wasm-bytecode - unless I misunderstood.

Reply:
You are right, let's keep it null